content/blog/todo.md

```diff
@@ -10,15 +10,42 @@ draft: false
 
 **TODO fix filename**
 
-I was recently at inception and I wanted to write down everything I learned.
+I recently spent a weekend in California at an Inception Studio event. For three days I hacked on large language model (LLM) ideas with some really smart people. I learned so much from the experience that I wanted to write down some of my biggest learnings.
```

Comment:
Is Inception Studio Event widely understood or should you define?

Reply:
Give a preview of article.  
I wanted to share  by big take aways:  Risk more; LLM insights; etc